security/securitytest/securitytest.go

```diff
@@ -17,25 +17,30 @@
 // Package securitytest embeds the TLS test certificates.
 package securitytest
 
-import "github.com/cockroachdb/cockroach/util"
+import (
+	"github.com/cockroachdb/cockroach/util"
+	"github.com/cockroachdb/cockroach/util/log"
+)
 
 //go:generate go-bindata -pkg securitytest -mode 0644 -modtime 1400000000 -o ./embedded.go -ignore README.md -prefix ../../resource ../../resource/test_certs/...
 //go:generate gofmt -s -w embedded.go
 //go:generate goimports -w embedded.go
 
-// TempRestrictedCopy creates a temporary on-disk copy of the embedded security asset
-// with the provided path. The copy will be created as a temporary file in the
-// provided directory; its filename will have the provided prefix. Returns the
-// path of the temporary file and a cleanup function that will delete the
-// temporary file.
+// RestrictedCopy creates an on-disk copy of the embedded security asset
+// with the provided path. The copy will be created in the provided directory.
+// Returns the path of the file and a cleanup function that will delete the file.
```

Comment:
I'm not sure what you mean. I renamed from "temporary file" to "file". It is still a file, so I'm not sure what I should change it to